src/plugins/scopes/ScopesPlugin.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+import BasePlugin from '../BasePlugin';
+import ScopesApi from './ScopesApi';
+import type Program from '../../elements/types/Program';
+import type Node from '../../elements/Node';
```

Comment:
Not sure if i can call this structure "variable", but yeah, you get the gist, naming things is hard